### PR TITLE
Don't show "no signal accounts" until contact intersection has completed

### PR DESCRIPTION
**Pull request by michaelkirk**
at least once

PTAL @charlesmchen

### CODE REVIEW

File: Signal/src/ViewControllers/MessageComposeTableViewController.m [hunk 1]
```diff
@@ -462,10 +463,15 @@ - (void)presentInviteFlow
...
-        [self showContactsPermissionReminder:NO];
+        if (self.contactsViewHelper.contactsManager.hasFetchedSignalAccountsAtLeastOnce
+            && self.contactsViewHelper.signalAccounts.count < 1
+            && ![[Environment preferences] hasDeclinedNoContactsView]) {
+            self.isNoContactsModeActive = YES;
+        } else {
+            self.isNoContactsModeActive = NO;
+        }
 
+        [self showContactsPermissionReminder:NO];
```
Comment (charlesmchen) [inline Signal/src/ViewControllers/MessageComposeTableViewController.m]: Shouldn't this read:

```
[self showContactsPermissionReminder:! self.isNoContactsModeActive];
```

Comment (charlesmchen) [inline Signal/src/ViewControllers/MessageComposeTableViewController.m]: Sorry this is wrong; ignore;

Comment (michaelkirk) [inline Signal/src/ViewControllers/MessageComposeTableViewController.m]: Well, you're not entirely wrong in that this method sucks.

**Comment by michaelkirk** [comment]
In 3ca2d0cbe0ee73637a69be0aa7ad9b86a5832edd

**Pull request closed without merging**